### ISSUE TITLE
Use UTC for RowKey and PartitionKey (in DateReverse mode)

### ISSUE THREAD

**Issue by erik-kallen** [stemarie/log4net.Azure#68]
I think it would be better if the RowKey and PartitionKey (in DateReverse mode) had been generated from UTC rather than from local time.

If you like the feature, I can probably do it myself and create a PR.

Thanks for a great utility, I really needed this one when I found it!

**Comment by snuffykl** [comment]
I agree to use UTC as TableStorage always store the data as UTC. Any plans to change it to UTC for RowKey and PartitionKey ?

**Comment by jpk951** [comment]
I agree on UTC as well.  Also, I'm having an issue with DateReverse in that the PartitionKey is the same for lot's of rows (like not granular enough?)

Oh, I see in code it's intentional for the PartitionKey to span an hour.  Thanks
